lib/gui/components/drive-selector/styles/_drive-selector.scss

```diff
@@ -35,3 +35,7 @@
   }
 }
 
+.modal-footer-warning {
+  text-align: center;
```

Comment:
You can do `@extend .text-center` here (the `.text-center` class is defined by Bootstrap)